assets/Templates/stratcards/base/0645434449AEC8D41E779DAB9C865D76.json

```diff
@@ -2,7 +2,7 @@
 	"Type": "Generic",
 	"GUID": "AE7D6A5C4EEDAAE2B2FEB5AFBA3AC6A1",
 	"Name": "Construction",
-	"Metadata": "",
+	"Metadata": "card.strategy/construction",

```

Comment:
I need to document the metadata namespace, this is actually "tile.strategy:base/construction".  I won't repeat this for most of the others, except where there's a notable tweak.